gtest-parallel

```diff
@@ -58,6 +58,9 @@ class SigintHandler(object):
   def __sigint_handler(self, signal_num, frame):
     with self.__lock:
       self.__on_sigint()
+  def interrupt(self):
```

Comment:
Yes, interrupt is a good name. But __sigint_handler is registered as a sigint handler, so if you eliminated it you'd have to... yes, you could register a lambda as sigint handler instead; the lambda would just ignore the signal_num and frame arguments and call interrupt(). Do that.